DXRando/DeusEx/Classes/DXRLamp2.uc

```diff
@@ -1,17 +1,61 @@
 //=============================================================================
-// Lamp2 except Warm White.  Fuck daylight bulbs.
+// Lamp2 except with a warm white light at the top.
+// Fuck daylight bulbs.
 // Localization files overwrite the ItemName,
 // so really force it to not be halogen
 //=============================================================================
 class DXRLamp2 injects #var(prefix)Lamp2;
 
+var DynamicLight lt;
+
+function InitLight()
+{
+    local vector loc;
+
+    if (lt == None) {
+        loc = Location;
+        loc.z += CollisionHeight * 1.35;
+
+        lt = Spawn(class'DynamicLight',,, loc);
+
+        lt.LightHue=44;
```

Comment:
what do you think about keeping these defined in defaultproperties and then doing

```suggestion
        lt.LightHue=default.LightHue;
```

Reply:
I'm very neutral on this, so sure. I suppose it would allow other classes to query this information if they wanted to for some reason.